extensions/amp-viewer-integration/amp-doc-viewer-api.md

```diff
@@ -0,0 +1,124 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AMP Viewer/Document API
```

Comment:
We should avoid slashes in document titles. I'd rename it `AMP document viewer API` or just `AMP viewer API`. Side note, is the AMP Viewer a proper noun, or is it AMP viewer?

Reply:
done. Amp viewer it is.